README.md

```diff
@@ -53,8 +50,7 @@ To contribute a patch, follow these instructions (note that installing
 ```
 
 
-License (see also LICENSE.txt)
-==============================
+## License (see also LICENSE.txt)
```

Comment:
hi @IMS94 I am saying that the rest of your heading formattings (for all else besides the first main heading) are also welcomed, so at a minimum:

1) remove them from this PR
2) if you like (and I am +1 for this so please do) open a new PR with *just those changes for the rest of the headings* and I will gladly commit it. 

HTH.